src/middleware/api.ts

```diff
@@ -101,3 +102,44 @@ export function hentEnhetTiltak(enhetId) {
 export function hentFeatures(featureQueryString: string) {
     return fetchToJson(`${API_BASE_URL}${FEATURE_URL}?${featureQueryString}`);
 }
+
+export function fetchHarSettInnlegg(): Promise<{Endringslogg: string}> | null {
+        return fetch(`${ENDRINGSLOGG_URL}?ressurs=Endringslogg`, {credentials: 'same-origin'})
+        .then(sjekkStatuskodeAndPost)
+        .then(toJson)
+        .catch(
+            () => {
+                return null;
+            }
+        );
+}
+
+function sjekkStatuskodeAndPost(response) {
+    response = sjekkStatuskode(response);
+    if(response.status === 204) {
+        return postNyRemoteStorage();
+    }
+    return response;
+}
+
+function postNyRemoteStorage(): Promise<Response> {
+    return fetch(ENDRINGSLOGG_URL, {
+        ...MED_CREDENTIALS,
+        method: 'POST',
+        body: JSON.stringify({})
```

Comment:
Trenger vi å sende med et tomt objekt? Eller brukes POSTen også for å overskrive hele objektet?